sycl-jit/jit-compiler/lib/translation/KernelTranslation.cpp

```diff
@@ -261,8 +260,9 @@ llvm::Expected<KernelBinary *> KernelTranslator::translateToPTX(
 
   // Give priority to user specified values (through environment variables:
   // SYCL_JIT_AMDGCN_PTX_TARGET_CPU and SYCL_JIT_AMDGCN_PTX_TARGET_FEATURES).
-  llvm::StringRef CPU{TargetCPU};
-  llvm::StringRef Features{TargetFeatures};
+  llvm::StringRef CPU = ConfigHelper::get<option::JITTargetCPU>().begin();
+  llvm::StringRef Features =
+      ConfigHelper::get<option::JITTargetFeatures>().begin();
```

Comment:
`ConfigHelper::get` returns a temporary, so `CPU` and `Features` contain dangling pointers, right?

Reply:
Yeap, this needs a bit more work, my build has tricked me again :) will ping you later.

Reply:
Thanks!

Reply:
I've given them a concrete storage of a `string` and fixed `DynArray` creation, should be OK now @victor-eds.